app/gui/view/src/popup.rs

```diff
@@ -0,0 +1,153 @@
+//! A temporary text message on top of the screen.
+
+use crate::prelude::*;
+
+use ensogl::animation::delayed::DelayedAnimation;
+use ensogl::application::Application;
+use ensogl::display;
+use ensogl::Animation;
+use ensogl_component::label::Label;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+const PADDING_TOP: f32 = 50.0;
+const DEFAULT_DELAY_MS: f32 = 5_000.0;
+
+
+
+// =============
+// === Model ===
+// =============
+
+/// Text label that disappears after a predefined delay.
+#[derive(Debug, Clone, CloneRef)]
+struct Model {
+    label:           Label,
+    network:         frp::Network,
+    delay_animation: DelayedAnimation,
+    /// Show the pop-up with the given message.
+    pub show:        frp::Source<String>,
+}
```

Comment:
This is a bit strange, that Model has it's FRP input (`show`) and `network`. The main reason we're making this models it to separate them from network (so capturing model won't leak any networks). The logic should be in View::new.

Reply:
Ah yes, I was trying to simplify the structure by not having a separate `PopupLabel` abstraction, this got a bit mixed. I'll fix it.